docs/pages/Overview.md

```diff
@@ -17,8 +17,46 @@ UI5 Tooling differentiates between "framework dependencies" and "project depende
 ## Linking Projects
 Would you like to work on an application project and one or more of its dependencies at the same time? We got you covered!
 
-By leaving dependency management up to the tool of your choice (see [FAQ: Why package.json? / Why npm?](./FAQ.md#why-packagejson-why-npm)) you have a variety of options.  
-Here is an example with [npm](https://www.npmjs.com/get-npm), an application, and a reuse library:
+### UI5 Workspaces
+The recommended approach for setting up a development environment where simultaneous work on multiple UI5 projects is required.
+
+Head over to the [UI5 Workspaces](./Workspace.md) documentation for all the details. Below is an example based on a simple scenario with an application and a reuse library project:
+
+**Example: Your Directory Structure**
+```
+my-app/
+    \_ node_modules/
+    \_ webapp/
+    \_ ui5.yaml
+    \_ package.json
+my-reuse-library/
+    \_ node_modules/
+    \_ src/
+    \_ test/
+    \_ ui5.yaml
+    \_ package.json
+```
+
+In its `package.json`, `my-app` should already define a dependency to `my-reuse-library`. So, after running the `npm install` command, a copy of the "my-reuse-library"-package should be retrieved from the package registry and added to my-app's `node_modules/` directory.
```

Comment:
should we add here a disclaimer regarding `npm install` in npm version 9 to add `--install-links false`?

Reply:
Thanks! But that option is actually only required when working with the `file://` protocol in dependencies (e.h. `"my-reuse-library": "file://../my-reuse-library"`. And even in that case, symlinks instead of copies are perfect since we can then omit the `npm link` step.

In this example however, we expect `my-reuse-library` to be a normal dependency stored in an npm registry. So npm install will just fetch a copy and store it in `node_modules/`. This behavior did not change with npm 9